src/eko/couplings.py

```diff
@@ -0,0 +1,729 @@
+        "decompose-expanded",
+        "perturbative-expanded",
+    ]:
+        return "expanded"
+    raise ValueError(f"Unknown evolution mode {mod_ev}")
+
+
+@nb.njit("f8[:](UniTuple(u1,2),f8[:],u1,f8,f8)", cache=True)
+def couplings_expanded(order, couplings_ref, nf, scale_from, scale_to):
+    """
+    Compute expanded expression.
+
+    Parameters
+    ----------
+        order: tuple(int, int)
+            perturbation order
+        couplings_ref: np array
+            reference alpha_s and alpha
+        nf: int
+            value of nf for computing the couplings
+        scale_from: float
+            reference scale
+        scale_to : float
+            target scale
+
+    Returns
+    -------
+        a : np array
+            couplings at target scale :math:`a(Q^2)`
+    """
+    # common vars
+    lmu = np.log(scale_to / scale_from)
+
+    def expanded_lo(ref, beta0, lmu):
+        den = 1.0 + beta0 * ref * lmu
+        return ref / den
+
+    def expanded_nlo(ref, beta0, beta1, lmu):
+        den = 1.0 + beta0 * ref * lmu
+        b1 = beta1 / beta0
+        as_LO = expanded_lo(ref, beta0, lmu)
+        as_NLO = as_LO * (1 - b1 * as_LO * np.log(den))
+        return as_NLO
+
+    def expanded_nnlo(ref, beta0, beta1, beta2, lmu):
+        as_LO = expanded_lo(ref, beta0, lmu)
+        as_NLO = expanded_nlo(ref, beta0, beta1, lmu)
+        b1 = beta1 / beta0
+        b2 = beta2 / beta0
+        res = as_LO * (
+            1.0
+            + as_LO * (as_LO - ref) * (b2 - b1**2)
+            + as_NLO * b1 * np.log(as_NLO / ref)
+        )
+        return res
+
+    def expanded_n3lo(ref, beta0, beta1, beta2, beta3, lmu):
+        b3 = b_qcd((3, 0), nf)
+        as_LO = expanded_lo(ref, beta0, lmu)
+        log_fact = np.log(as_LO)
+        b1 = beta1 / beta0
+        b2 = beta2 / beta0
+        b3 = beta3 / beta0
+        res = expanded_nnlo(ref, beta0, beta1, beta2, lmu)
+        res += (
+            as_LO**4
+            / (2 * beta0**3)
+            * (
+                -2 * b1**3 * np.log(ref) ** 3
+                + 5 * b1**3 * log_fact**2
+                + 2 * b1**3 * log_fact**3
+                + b1**3 * np.log(ref) ** 2 * (5 + 6 * log_fact)
+                + 2
+                * beta0
+                * b1
+                * log_fact
+                * (b2 + 2 * (b1**2 - beta0 * b2) * lmu * ref)
+                - beta0**2
+                * lmu
+                * ref
+                * (
+                    -2 * b1 * b2
+                    + 2 * beta0 * b3
+                    + (b1**3 - 2 * beta0 * b1 * b2 + beta0**2 * b3) * lmu * ref
+                )
+                - 2
+                * b1
+                * np.log(ref)
+                * (
+                    5 * b1**2 * log_fact
+                    + 3 * b1**2 * log_fact**2
+                    + beta0 * (b2 + 2 * (b1**2 - beta0 * b2) * lmu * ref)
+                )
+            )
+        )
+        return res
+
+    if order[1] == 0:
```

Comment:
is there anything shared between the (if-) branches? if not make them a seperate function each

Reply:
No it's not. It's because previously the names where given factorizing a term as1 from beta_qcd and aem1 from beta_qed but then i reinserted it but i forgot to change the names inside the functions. I'm going to fix it